.github/workflows/ping-other-repos.yml

```diff
@@ -0,0 +1,25 @@
+name: Remote Dispatch Action Initiator
+
+on:
+  push:
+  repository_dispatch:
+
+jobs:
+  ping-pong:
+    runs-on: ubuntu-latest
+    steps:
+      - name: PING - Dispatch initiating repository event
+        if: github.event.action != 'pong'
```

Comment:
Can we guard against the secret being unset as well? Otherwise, every fork will attempt to complete this step and fail, because it doesn't have the right permissions.

Reply:
> Can we guard against the secret being unset as well? Otherwise, every fork will attempt to complete this step and fail, because it doesn't have the right permissions.

Good point. Added a conditional for it.

And note that per https://github.com/actions/runner/issues/520:
> Unfortunately, GitHub Actions currently does not support checking the presence of a secret in a conditional

So, as suggested in https://github.com/actions/runner/issues/520#issuecomment-860043020, I did it by copying the secret to an environment variable first, then checking if that environment variable is set.

Reply:
> Can we guard against the secret being unset as well? Otherwise, every fork will attempt to complete this step and fail, because it doesn't have the right permissions.

On second thought, looking back at the source, it already has this:
```yaml
if: ${{ github.respository == 'mdn/browser-compat-data' }}
```
So, would that not be sufficient for ensuring it doesn’t run for forks?